release-notes/unreleased/2496-wrong-dst.yml

```diff
@@ -0,0 +1,8 @@
+issue_key: 2438
+show_in_stores: true
+platforms:
+  - android
+  - ios
+  - web
+en: Fixed wrong dates caused by daylight savings time.
+de: Falsch angezeigte Daten durch Zeitverschiebung behoben.
```

Comment:
This translation is nor really correct. "daylaight saving time" is "Sommerzeit". "time shift" would be "Zeitverschiebung".

Reply:
Will use `Zeitumstellung`.